modules/core/shared/src/main/scala/lucuma/core/math/package.scala

```diff
@@ -40,3 +40,29 @@ type LineFluxValue = LineFluxValue.Type
 type FluxDensityContinuumValueRefinement = Interval.Closed[0, 1]
 object FluxDensityContinuumValue extends RefinedNewType[BigDecimal, FluxDensityContinuumValueRefinement]
 type FluxDensityContinuumValue = FluxDensityContinuumValue.Type
+
+// Copied from:
+// https://github.com/ghewgill/picomath/blob/master/scala/src/Erf.scala
+object Erf:
```

Comment:
`Erf` is new to me.

Reply:
Me too, luckily somebody had done it before